src/main/java/seedu/mtracker/console/AddCryptoParser.java

```diff
@@ -30,17 +48,33 @@ public void addCryptoExpiryToParameters() throws OperationAbortedError {
         AssertParserHelper.assertExpiryInTheFuture(expiry);
     }
 
+    /**
+     * Gets the user crypto remarks input and adds it into the parameters list.
+     *
+     * @throws OperationAbortedError If the user wants to abort the add crypto process.
+     */
     public void addCryptoRemarksToParameters() throws OperationAbortedError {
         String remarks = getCryptoRemarksFromUser();
         checkIfAbort(remarks, WORKSPACE);
         parameters.add(remarks);
     }
 
+    /**
+     * Gets from the user parameters that are specific to adding a new crypto instrument.

```

Comment:
Change to "Gets crypto specific parameters from the user when adding a new crypto instrument"